From 4de1f70135fa4b44eb5016e36ba08ffd0bc62414 Mon Sep 17 00:00:00 2001
From: Wen He <wen.he_1@nxp.com>
Date: Wed, 27 Nov 2019 17:48:17 +0800
Subject: [PATCH] arm64: dts: ls1028a: Add DP DT nodes

Add DP DT nodes for configure and enable the HD Display
controller on LS1028ARDB and LS1028AQDS boards.

Signed-off-by: Wen He <wen.he_1@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts | 9 +--------
 arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts | 9 +--------
 2 files changed, 2 insertions(+), 16 deletions(-)

--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
@@ -274,14 +274,7 @@
 };
 
 &hdptx0 {
-	fsl,no_edid;
-	resolution = "3840x2160@60",
-		   "1920x1080@60",
-		   "1280x720@60",
-		   "720x480@60";
-	lane_mapping = <0x4e>;
-	edp_link_rate = <0x6>;
-	edp_num_lanes = <0x4>;
+	lane-mapping = <0x4e>;
 	status = "okay";
 };
 
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
@@ -262,13 +262,6 @@
 };
 
 &hdptx0 {
-	fsl,no_edid;
-	resolution = "3840x2160@60",
-		   "1920x1080@60",
-		   "1280x720@60",
-		   "720x480@60";
-	lane_mapping = <0x4e>;
-	edp_link_rate = <0x6>;
-	edp_num_lanes = <0x4>;
+	lane-mapping = <0x4e>;
 	status = "okay";
 };
